spec/models/chess_piece_spec.rb

```diff
@@ -140,5 +140,23 @@
     end
   end
 
+  describe "#capture" do
+    context "given a black piece at [4,4] and a white piece at [5,5]" do
+      let(:game) {Game.create}
+      let(:piece) {ChessPiece.create(position_x: 4, position_y: 4, game: game, color: "Black")}
+      let(:second_piece) {ChessPiece.create(position_x: 5, position_y: 5, game: game, color: "White")}
+      it "will return true when trying to move to [5,5] if both pieces are different colors" do
+        if piece.color != second_piece.color
```

Comment:
@KaoruDev Ok, I think I see what you're getting at. Because `piece` and `second_piece` have defined colors, `piece.color != second_piece.color` is always true.